addons/mail/data/mail_templates_email_layouts.xml

```diff
@@ -8,57 +8,60 @@
 </head>
 <body style="font-family:Verdana, Arial,sans-serif; color: #454748;">
 <t t-set="subtype_internal" t-value="subtype and subtype.internal"/>
+<t t-set="show_header" t-value="has_button_access or subtype_internal"/>
```

Comment:
Would allow forcing it if necessary
```suggestion
<t t-set="show_header" t-value="has_button_access or subtype_internal or force_show_header"/>
```